bitrise.yml

```diff
@@ -168,16 +168,17 @@ workflows:
         - xctestrun: $BITRISE_TEST_BUNDLE_PATH/Fennec_UnitTest_iphonesimulator18.2-arm64.xctestrun
         - maximum_test_repetitions: 2
     - xcode-test-shard-calculation@0:
+        run_if: '{{enveq "BITRISE_TEST_BUNDLE_PATH" "" | not}}'
```

Comment:
This is a env variable created when xcode-build-for testing passes. If there is a compilation error, this will be empty and this steps, as well as deploy step will be skipped